easybuild/easyconfigs/t/tlparse/tlparse-0.3.5-GCCcore-13.2.0.eb

```
@@ -0,0 +1,42 @@
+easyblock = 'PythonPackage'
+
+name = 'tlparse'
+version = '0.3.5'
+
+homepage = 'https://github.com/ezyang/tlparse'
+description = "Parse structured PT2 (PyTorch 2) logs"
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+github_account = 'ezyang'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['256eb651bc435c75ff33ad4a31695db99e7e8441d45d2a3124c66fbe2a4b1d30']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('maturin', '1.5.0', '-Rust-1.76.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+]
+
+use_pip = True
+sanity_pip_check = False
```

CI complains about this. It seems this doesn't really install a Python package but rather a binary only it doesn't make much sense enabling this, but it doesn't hurt either as with it there won't be any failures that weren't there before, will they?

---

With it set to True it expects there to be a lib/pythonx.y/... directory.

One might be able to use the cargo build instead but I haven't looked at that yet

---

Where is that happening? The code for `sanity_pip_check` is https://github.com/easybuilders/easybuild-easyblocks/blob/0b5f61d5b31157b67169fb483546aab8fd6e31fc/easybuild/easyblocks/generic/pythonpackage.py#L1012-L1082

I don't see any reference to `lib/python*`. I also tried to install it with this set to True and didn't find any issues.

What I found: If it is not an extension and no sanity_check_paths or custom_paths have been set in the EC it will check that dir. And it IS there: `tlparse/0.3.5-GCCcore-13.2.0/lib/python3.11/site-packages/tlparse-0.3.5.dist-info`

So I don't see any issues:
```suggestion
sanity_pip_check = True
```

---

Yeah, it might have been that it tried to load a tlparse module. I turned it off when setting things up and forgot to re-enable it after having set modulename: False
Fixed now anyway